packages/bson-bench/test/unit/suite.test.ts

```diff
@@ -0,0 +1,115 @@
+import { expect } from 'chai';
+import { readFile } from 'fs/promises';
+
+import { Suite } from '../../lib';
+import { exists } from '../../src/utils';
+import { clearTestedDeps } from '../utils';
+
+describe('Suite', function () {
+  beforeEach(clearTestedDeps);
+  after(clearTestedDeps);
+
+  describe('#constructor()', function () {});
+  describe('#task()', function () {
+    it('returns the Suite it was called on', function () {
+      const suite = new Suite('test');
+      expect(
+        suite.task({
+          documentPath: 'test',
+          operation: 'serialize',
+          warmup: 10,
+          iterations: 10,
+          library: 'bson@6.0.0',
+          options: {}
+        })
+      ).to.equal(suite);
+    });
+  });
```

Comment:
```suggestion

  describe('#task()', function () {
    it('returns the Suite it was called on', function () {
      const suite = new Suite('test');
      expect(
        suite.task({
          documentPath: 'test',
          operation: 'serialize',
          warmup: 10,
          iterations: 10,
          library: 'bson@6.0.0',
          options: {}
        })
      ).to.equal(suite);
    });
  });
```

Spacing for readability.